src/normalwishart.jl

```diff
@@ -40,7 +40,7 @@ end
 function insupport(::Type{NormalWishart}, x::Vector{T}, Lam::Matrix{T}) where T<:Real
     return (all(isfinite(x)) &&
            size(Lam, 1) == size(Lam, 2) &&
-           isApproxSymmmetric(Lam) &&
+          #isApproxSymmmetric(Lam) &&
```

Comment:
This (and `hasCholesky`) are in Distributions, so can be fixed by qualifying with `Distributions.`

Reply:
maybe worth moving them over here since I don't think they're used in Distributions anymore...